controllers/update_test.go

```diff
@@ -388,6 +390,162 @@ Images:
 		})
 	})
 
+	Context("commit signing", func() {
+
+		var localRepo *git.Repository
+
+		// generate keypair for signing
+		pgpEntity, err := openpgp.NewEntity("", "", "", nil)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
This won't work as you expect outside of BeforeEach/AfterEach/It. Make pgpEntity a variable and initialise it in `BeforeEach`.

Reply:
I'm not quite sure how the test still passes -- maybe because it doesn't matter how it gets initialised, so long as it does once. In any case, it would be neater to keep initialisation inside `BeforeEach`.

Reply:
Good catch, have shifted this around